avm/res/event-grid/system-topic/event-subscription/main.bicep

```
@@ -72,7 +72,7 @@ resource eventSubscription 'Microsoft.EventGrid/systemTopics/eventSubscriptions@
     deadLetterDestination: !empty(deadLetterDestination) ? deadLetterDestination : null
     deadLetterWithResourceIdentity: !empty(deadLetterWithResourceIdentity) ? deadLetterWithResourceIdentity : null
     deliveryWithResourceIdentity: !empty(deliveryWithResourceIdentity) ? deliveryWithResourceIdentity : null
-    destination: destination
+    destination: !empty(deliveryWithResourceIdentity) ? null: destination
```

```suggestion
    destination: empty(deliveryWithResourceIdentity) ? destination : null
```
Always a fan of having the `null` as an else as it's easier to read. Note - requires a regeneration of the `main.json`

---

OK !